### DISCUSSION TITLE
Tarteaucitron et Matomo Tag Manager

### DISCUSSION THREAD

**Discussion by Folofietre** [AmauriC/tarteaucitron.js#1009] [Q&A] [answered]
Bonjour ! 

J'ai utilisé avec succès Tarteaucitron et Matomo sans aucun soucis, et j'essaie maintenant de configurer un Matomo Tag Manager.

J'ai bien ajouté le script :
`<script type="text/javascript">
        tarteaucitron.user.matomotmUrl = 'matomomtUrl';
        (tarteaucitron.job = tarteaucitron.job || []).push('matomotm');
</script>
`

Dans mon inspecteur j'ai bien ce script avec mon url. Néanmoins je n'ai aucun cookie créé et rien dans le panneau de gestion des cookies. 
Sauriez vous m'aiguiller vers une solution ?

Merci beaucoup !

**Comment by AmauriC** [comment, marked as answer]
Hello
Possible d'avoir l'url du site? Les fichiers tarteaucitron sont bien à jour?

**Reply by Folofietre** [reply]
Je viens de me rendre compte que la version utilisée est la 1.9.6, je vais tenter une mise à jour !

EDIT : Effectivement, en passant à la 1.9.9 Matomo Tag Manager apparait bien ! Merci beaucoup et désolé du dérangement !
